README.md

```diff
@@ -16,9 +16,15 @@ There are also some CSS frameworks like Bootstrap [that is broadly used in ~30%
 
 ## 🌱  How to start this project
 
-1. This project comes with the necessary files to start working.
-2. Clone this repository locally or open it with gitpod for a better experience.
-3. Run the following command to preview your changes: `npx --yes http-server`
+This project comes with the necessary files to start working, but you have two options to start:
+
+a) Use Gitpod: copy & paste this on your browser: gitpod.io#<https://github.com/breatheco-de/exercise-product-landing-page-plain-css.git>
+
+b) You can clone this repository on your local computer:
+```sh
+$ git clone https://github.com/breatheco-de/exercise-product-landing-page-plain-css.git
+```
+**Remember to run the following command to preview your changes: `npx --yes http-server`**
```

Comment:
Make sure that you have node.js installed in your computer and run the following command on your terminal to preview your website:
```bash
npx --yes http-server
```